### PR TITLE
Feat/various

### PR DESCRIPTION
**Pull request by streamich**
- [x] Logo
  - [x] add `semantic-release`
  - [x] `<Prompt>`
  - [x] `<Alert>`
  - [X] `<ViewportSensor>`
  - [X] `<OrientationSensor>` using [`orientationchange` event](https://developer.mozilla.org/en-US/docs/Web/Events/orientationchange)
  - [X] Add Travic CI

### CODE REVIEW

File: docs/OrientationSensor.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# `<OrientationSensor>`
+
+Tracs screen orientation using [`orientationchange` event](https://developer.mozilla.org/en-US/docs/Web/Events/orientationchange).
```
Comment (xram) [inline docs/OrientationSensor.md]: `Tracks`

File: docs/Prompt.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  onResult={console.log}
+>{(result) => <div>{result}</div>}</Prompt>
+```
+
+## Props
+
+  - `show` - boolean, optional, whether to show the prompt modal.
+  - `message` - string, optional, string message to display to the user.
+  - `default` - string, optional, default text to pre-fill the user's response input.
+  - `onResult` - function, optional, function that receives propt result string as a single argument.
```
Comment (xram) [inline docs/Prompt.md]: `prompt`

File: docs/ViewportSensor.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  threshold?: number;
+  onChange?: (state: IViewportObserverSensorState) => void;
+}
+```
+
+, where
+
+  - `threshold` - optional, number, percentage how much does your component have to intersect with viewport
+  to be considered visible. Defaults to `0`.
+  - `onChange` - optional, callback called when sensor changes it's state, receives the state of the sensor as
```
Comment (xram) [inline docs/ViewportSensor.md]: `its` (no apostrophe).

File: docs/getDisplayName.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+Returns display name of a React component.
+
+```jsx
+import getDisplayName from 'libreact/lib/util/getDisplayName';
+
+const name = getDisplayName(MyComponent);
+```
+
+Accepts a single argument, which can be a statefull or stateless React component, or element. Returns a string.
```
Comment (xram) [inline docs/getDisplayName.md]: `stateful`

File: src/ViewportObserverSensor/index.ts [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+  ref = (originalRef) => (el) => {
+    this.el = el;
+    (originalRef || noop)(el);
+  };
+
+  componentDidMount () {
+    const {margin, threshold} = this.props;
+
+    this.mounted = true;
+console.log('MAR', margin.map((val) => val + 'px').join(' '));
```
Comment (xram) [inline src/ViewportObserverSensor/index.ts]: Please remove.

File: src/ViewportScrollSensor/index.ts [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+  const y6 = Math.min(y2, y4);
+
+  if ((x5 >= x6) || (y5 >= y6)) {
+    return null;
+  }
+
+  return [x5, y5, x6, y6];
+};
+
+// Compute are of a NORMALIZED rectangle.
```
Comment (xram) [inline src/ViewportScrollSensor/index.ts]: `area`

File: docs/OrientationSensor.md (hunk 1 above)
Comment (streamich) [reply docs/OrientationSensor.md]: Done.

File: docs/Prompt.md (hunk 2 above)
Comment (streamich) [reply docs/Prompt.md]: Done.

File: docs/ViewportSensor.md (hunk 3 above)
Comment (streamich) [reply docs/ViewportSensor.md]: Done.

File: docs/getDisplayName.md (hunk 4 above)
Comment (streamich) [reply docs/getDisplayName.md]: Done.

File: src/ViewportObserverSensor/index.ts (hunk 5 above)
Comment (streamich) [reply src/ViewportObserverSensor/index.ts]: Done.

File: src/ViewportScrollSensor/index.ts (hunk 6 above)
Comment (streamich) [reply src/ViewportScrollSensor/index.ts]: Done.

**Pull request merged**